test/Integration/vectorization-mlp.mlir

```diff
@@ -0,0 +1,5 @@
+// RUN: export vectorFile=$(mktemp)
+// RUN: mlir-gen   --kernel=const --bias --relu --layers=16,16 --tiles=4,4,4 -float-type=bf16 --vnni=2  | tpp-run -linalg-to-vector -e entry --entry-point-result=void -print -n 1 --seed=123 2>&1 > $vectorFile
```

Comment:
nit: maybe this shouldn't use `--vnni` flag to also check correct layout changes